src/mmw/apps/geoprocessing_api/views.py

```diff
@@ -1106,8 +1107,86 @@ def start_analyze_pointsource(request, format=None):
     area_of_interest, wkaoi, msg = _parse_analyze_input(request,
                                                         perimeter='CONUS')
 
+    deprecation_message = [
```

Comment:
Let's append this to `msg` which may have other messages:


```suggestion
    msg.append(
```